dashboard/config/locales/scripts.en.yml

```diff
@@ -6285,7 +6285,7 @@ en:
         hello-world-draft:
           lesson_groups: {}
           name: hello-world-draft
-          title: '"Hello, World!"'
+          title: Hello, World!
```

Comment:
Is there a way to make sure this problem doesn't happen in the future?

Reply:
yep, this is the future work:

> We should be canonicalizing the S3 filenames, so that when we upload to a filename, we can reliably check whether that file exists later: [PLAT-1925](https://codedotorg.atlassian.net/browse/PLAT-1925)

"canonicalize" in this case means replacing any special characters with `-`, so that when the file gets uploaded, the name we specify is the one that will actually get written to S3.

Reply:
`"` would be on the of the characters we would replace with `-`.

Reply:
Oh got it! Thanks